app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_ground_truth.html

```diff
@@ -16,49 +16,85 @@
 {% endblock %}
 
 {% block content %}
-    <h2>Ground Truth</h2>
+    <h2>Ground Truth of {{ object }}</h2>
 
     <p>
-        If you wish to assess the performance of the readers you
-        can upload the CSV file containing the ground truth for
-        this reader study here.
+        A ground truth consists of the correct answers to the questions for each case, it serves as a
```

Comment:
Ground Truth would not be singular.

Reply:
T.b.h. I am a bit confused on the correct usage of singular and plural forms with respect to 'ground truth' or akin collections. Likely too subtle for a brute-force Dutchie!

When referencing it as a singular collection I'd be inclined to use 'ground truth'. I am not sure when or where to use the plural 'ground truths'. Or should perhaps the problem lies with the 'A ...' and it should be 'The ground truth...'?

Reply:
I would say The Ground Truth, A Ground Truth sounds wrong to me but unfortunately I cannot explain why - the style of English education at the time was to do away with rules.

Reply:
Roger! Let us then fully lean on your English roots! Thank you for noticing.

On the uptick, I did find a misspelling of 'wish' as 'widh' that was part of the CSV paragraphs for the longest time!